### PR TITLE
Add elementwise math operations

### PR DESCRIPTION
**Pull request by emailweixu**
This allows user to use expressions like: y=log(1+exp(x))

Also added unittests for ActivationFunction

### CODE REVIEW

File: python/paddle/trainer_config_helpers/math.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                           name=name,
+                           act=act)
+    op = wrap_name_default(op_name)(op)
+    op.__doc__ = type(act).__doc__
+    globals()[op_name]= op
+    __all__.append(op_name)
+
+# register_unary_math_op('log', act.LogActivation())
+register_unary_math_op('exp', act.ExpActivation())
+register_unary_math_op('log', act.ExpActivation())
```
Comment (luotao1) [inline python/paddle/trainer_config_helpers/math.py]: line 35 is `register_unary_math_op('log', act.LogActivation())` ?

**Comment by luotao1** [comment]
Can add LogActivation in `doc/ui/api/trainer_config_helpers/activations.rst` ?

**Comment by reyoung** [comment]
It seems @luotao1 has been reviewed this PR. So I change assignees to luotao01

File: python/paddle/trainer_config_helpers/activations.py [hunk 2]
```diff
@@ -199,3 +199,12 @@ class ExpActivation(BaseActivation):
        f(z) = e^z.
     """
     def __init__(self): BaseActivation.__init__(self, 'exponential', False)
+
+class LogActivation(BaseActivation):
+    """
+    Exponential Activation.
```
Comment (luotao1) [inline python/paddle/trainer_config_helpers/activations.py]: line 205 is Log Activation

**Pull request merged**